pages/integrations/git.md.erb

```diff
@@ -0,0 +1,92 @@
+# Git Server-Side Hooks
+
+You can trigger builds using Git hook scripts and the Buildkite REST API.
+
+This page explains how to trigger builds when you push to a Git server.
+For example, if you're using a proprietary Git server, then you can trigger builds on push with a post-recieve hook.
+This method can be adapted for other Git events or for running Buildkite builds from arbitrary scripts and services.
+
+{:toc}
+
+## Before you start
```

Comment:
> Before you start

OK, this is strictly speaking unconventional — "prerequisites" seems to be the done thing around here — but it doesn't have to be. Just saying.

Reply:
I count 3 `prerequisites` and 1 `before you start`, we can easily make this change.
I think often we just put it in body text, but I'm +1 on being explicit.

Reply:
Replaced the other 3 with ee886671. Can cherry-pick to a separate PR if this unwanted here.